diff --git a/static/admin/js/SelectBox.js b/static/admin/js/SelectBox.js
--- a/static/admin/js/SelectBox.js
+++ b/static/admin/js/SelectBox.js
@@ -7,1 +7,1 @@
-            SelectBox.cache[id] = [];
+      SelectBox.cache[id] = [];
@@ -16,1 +16,1 @@
-            const scroll_value_from_top = box.scrollTop;
+      const scroll_value_from_top = box.scrollTop;
@@ -38,1 +38,1 @@
-                        break; // Once the first token isn't found we're done
+            break; // Once the first token isn't found we're done
@@ -54,1 +54,1 @@
-                    break;
+          break;
diff --git a/static/admin/js/cancel.js b/static/admin/js/cancel.js
--- a/static/admin/js/cancel.js
+++ b/static/admin/js/cancel.js
@@ -5,1 +5,1 @@
-    // http://youmightnotneedjquery.com/#ready
+  // http://youmightnotneedjquery.com/#ready
diff --git a/static/admin/js/change_form.js b/static/admin/js/change_form.js
--- a/static/admin/js/change_form.js
+++ b/static/admin/js/change_form.js
@@ -6,1 +6,1 @@
-        const form = document.getElementById(modelName + '_form');
+    const form = document.getElementById(modelName + "_form");
diff --git a/static/admin/js/core.js b/static/admin/js/core.js
--- a/static/admin/js/core.js
+++ b/static/admin/js/core.js
@@ -11,1 +11,1 @@
-    const len = arguments.length;
+  const len = arguments.length;
@@ -89,1 +89,1 @@
-        return typeof window.CalendarNamespace === "undefined"
+    return typeof window.CalendarNamespace === "undefined"
@@ -90,1 +90,1 @@
-            ? this.getTwoDigitMonth()
+      ? this.getTwoDigitMonth()
diff --git a/static/admin/js/SelectFilter2.js b/static/admin/js/SelectFilter2.js
--- a/static/admin/js/SelectFilter2.js
+++ b/static/admin/js/SelectFilter2.js
@@ -31,1 +31,1 @@
-            // <div class="selector"> or <div class="selector stacked">
+      // <div class="selector"> or <div class="selector stacked">
@@ -44,1 +44,1 @@
-                        'This is the list of available %s. You may choose some by ' +
+            "This is the list of available %s. You may choose some by " +
diff --git a/static/admin/js/collapse.js b/static/admin/js/collapse.js
--- a/static/admin/js/collapse.js
+++ b/static/admin/js/collapse.js
@@ -4,1 +4,1 @@
-    window.addEventListener('load', function() {
+  window.addEventListener("load", function () {
diff --git a/static/drf-yasg/swagger-ui-init.js b/static/drf-yasg/swagger-ui-init.js
--- a/static/drf-yasg/swagger-ui-init.js
+++ b/static/drf-yasg/swagger-ui-init.js
@@ -14,1 +14,1 @@
-        .replace(/[^\w\-]+/g, '')       // Remove all non-word chars
+        .replace(/[^\w-]+/g, '')       // Remove all non-word chars
@@ -132,1 +132,1 @@
-        if (swaggerSettings.hasOwnProperty(p)) {
+        if (Object.prototype.hasOwnProperty.call(swaggerSettings, p)) {
diff --git a/static/drf-yasg/immutable.js b/static/drf-yasg/immutable.js
--- a/static/drf-yasg/immutable.js
+++ b/static/drf-yasg/immutable.js
@@ -34,1 +34,1 @@
-      return isIndexed(value) ? value : IndexedSeq(value);
+    return isIndexed(value) ? value : IndexedSeq(value);
@@ -409,1 +409,1 @@
-      return this._object.hasOwnProperty(key);
+      return Object.prototype.hasOwnProperty.call(this._object, key);
diff --git a/static/drf-yasg/insQ.js b/static/drf-yasg/insQ.js
--- a/static/drf-yasg/insQ.js
+++ b/static/drf-yasg/insQ.js
@@ -6,1 +6,1 @@
-        animation_string = 'animationName',
+    animation_string = "animationName",
@@ -152,1 +152,1 @@
-            if (opt.hasOwnProperty(o)) {
+            if (Object.prototype.hasOwnProperty.call(opt, o)) {
diff --git a/static/admin/js/admin/RelatedObjectLookups.js b/static/admin/js/admin/RelatedObjectLookups.js
--- a/static/admin/js/admin/RelatedObjectLookups.js
+++ b/static/admin/js/admin/RelatedObjectLookups.js
@@ -8,1 +8,1 @@
-    const relatedWindows = [];
+  const relatedWindows = [];
diff --git a/static/admin/js/inlines.js b/static/admin/js/inlines.js
--- a/static/admin/js/inlines.js
+++ b/static/admin/js/inlines.js
@@ -45,1 +45,1 @@
-         * The "Add another MyModel" button below the inline forms.
+     * The "Add another MyModel" button below the inline forms.
@@ -52,1 +52,1 @@
-                    const numCols = $this.eq(-1).children().length;
+          const numCols = $this.eq(-1).children().length;
@@ -65,1 +65,1 @@
-            e.preventDefault();
+      e.preventDefault();
@@ -67,1 +67,1 @@
-            const row = template.clone(true);
+      const row = template.clone(true);
diff --git a/static/admin/js/calendar.js b/static/admin/js/calendar.js
--- a/static/admin/js/calendar.js
+++ b/static/admin/js/calendar.js
@@ -12,1 +12,1 @@
-            gettext('February'),
+      gettext("February"),
@@ -28,1 +28,1 @@
-            pgettext('abbrev. month April', 'Apr'),
+      pgettext("abbrev. month April", "Apr"),
@@ -57,1 +57,1 @@
-                days = 30;
+        days = 30;
diff --git a/static/rest_framework/docs/js/api.js b/static/rest_framework/docs/js/api.js
--- a/static/rest_framework/docs/js/api.js
+++ b/static/rest_framework/docs/js/api.js
@@ -3,1 +3,1 @@
-var schema = window.schema
+var schema = window.schema;
@@ -52,1 +52,1 @@
-        entries.push([element.name, element.selectedOptions[j].value])
+        entries.push([element.name, element.selectedOptions[j].value]);
@@ -148,1 +148,1 @@
-      if (!params.hasOwnProperty(name)) {
+      if (!Object.prototype.hasOwnProperty.call(params, name)) {
diff --git a/static/drf-yasg/redoc-init.js b/static/drf-yasg/redoc-init.js
--- a/static/drf-yasg/redoc-init.js
+++ b/static/drf-yasg/redoc-init.js
@@ -37,1 +37,1 @@
-    if (redocSettings.hasOwnProperty(p)) {
+    if (Object.prototype.hasOwnProperty.call(redocSettings, p)) {
diff --git a/static/admin/js/autocomplete.js b/static/admin/js/autocomplete.js
--- a/static/admin/js/autocomplete.js
+++ b/static/admin/js/autocomplete.js
@@ -24,1 +24,1 @@
-    $(function() {
+  $(function () {
diff --git a/static/admin/js/theme.js b/static/admin/js/theme.js
--- a/static/admin/js/theme.js
+++ b/static/admin/js/theme.js
@@ -7,1 +7,1 @@
-                console.error(`Got invalid theme mode: ${mode}. Resetting to auto.`);
+        console.error(`Got invalid theme mode: ${mode}. Resetting to auto.`);
@@ -9,1 +9,1 @@
-            }
+      }
diff --git a/static/rest_framework/js/ajax-form.js b/static/rest_framework/js/ajax-form.js
--- a/static/rest_framework/js/ajax-form.js
+++ b/static/rest_framework/js/ajax-form.js
@@ -62,1 +62,1 @@
-    contentType = form.attr('enctype') || form.attr('encoding')
+    contentType = form.attr("enctype") || form.attr("encoding");
diff --git a/static/admin/js/nav_sidebar.js b/static/admin/js/nav_sidebar.js
--- a/static/admin/js/nav_sidebar.js
+++ b/static/admin/js/nav_sidebar.js
@@ -20,1 +20,1 @@
-            localStorage.setItem('django.admin.navSidebarIsOpen', navSidebarIsOpen);
+      localStorage.setItem("django.admin.navSidebarIsOpen", navSidebarIsOpen);
@@ -33,1 +33,1 @@
-            options.push({title: container.innerHTML, node: container});
+      options.push({ title: container.innerHTML, node: container });
@@ -41,1 +41,1 @@
-            if (event.key === 'Escape') {
+      if (event.key === "Escape") {
diff --git a/static/rest_framework/js/default.js b/static/rest_framework/js/default.js
--- a/static/rest_framework/js/default.js
+++ b/static/rest_framework/js/default.js
@@ -17,1 +17,1 @@
-    $(e.target).parents('.tabbable').removeClass('first-tab-active');
+    $(e.target).parents(".tabbable").removeClass("first-tab-active");
@@ -45,1 +45,1 @@
-    $('#errorModal').modal('show');
+    $("#errorModal").modal("show");
diff --git a/static/rest_framework/js/csrf.js b/static/rest_framework/js/csrf.js
--- a/static/rest_framework/js/csrf.js
+++ b/static/rest_framework/js/csrf.js
@@ -38,1 +38,1 @@
-    !(/^(\/\/|http:|https:).*/.test(url));
+    !(/^(?:\/\/|http:|https:).*/.test(url));
diff --git a/static/admin/js/filters.js b/static/admin/js/filters.js
--- a/static/admin/js/filters.js
+++ b/static/admin/js/filters.js
@@ -25,1 +25,1 @@
-        detail.addEventListener('toggle', event => {
+    detail.addEventListener("toggle", (event) => {
diff --git a/static/admin/js/prepopulate.js b/static/admin/js/prepopulate.js
--- a/static/admin/js/prepopulate.js
+++ b/static/admin/js/prepopulate.js
@@ -2,1 +2,1 @@
-'use strict';
+"use strict";
@@ -38,1 +38,1 @@
-            if (!prepopulatedField.val()) {
+      if (!prepopulatedField.val()) {
diff --git a/static/admin/js/actions.js b/static/admin/js/actions.js
--- a/static/admin/js/actions.js
+++ b/static/admin/js/actions.js
@@ -47,1 +47,1 @@
-        if (checked) {
+    if (checked) {
@@ -143,1 +143,1 @@
-        };
+        }
diff --git a/static/admin/js/admin/DateTimeShortcuts.js b/static/admin/js/admin/DateTimeShortcuts.js
--- a/static/admin/js/admin/DateTimeShortcuts.js
+++ b/static/admin/js/admin/DateTimeShortcuts.js
@@ -5,1 +5,1 @@
-'use strict';
+"use strict";
@@ -18,1 +18,1 @@
-            ]
+      ],
@@ -30,1 +30,1 @@
-        init: function() {
+    init: function () {
@@ -32,1 +32,1 @@
-            if (serverOffset) {
+      if (serverOffset) {
@@ -48,1 +48,1 @@
-        // Return the current time while accounting for the server timezone.
+    // Return the current time while accounting for the server timezone.
@@ -53,1 +53,1 @@
-                const localOffset = localNow.getTimezoneOffset() * -60;
+        const localOffset = localNow.getTimezoneOffset() * -60;